src/chart_types/xy_chart/renderer/dom/_screen_reader.scss

```
@@ -0,0 +1,7 @@
+.screen-reader {
+  position: absolute;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
```

Does this need to be visible to work with the screen reader? we could add `pointer-events: none` and `visibility: hidden`?

```suggestion
  position: absolute;
  pointer-events: none;
  visibility: hidden;
  width: 1px;
  height: 1px;
```

---

Hiding stuff has historically been fraught with bugs so it's best avoided at the time being. 

This almost matches [EUI's screen reader only mixin](https://github.com/elastic/eui/blob/master/src/global_styling/mixins/_helpers.scss#L90-L97) which has been battle tested for a while now.

The only thing that EUI has that this is missing is `overflow: hidden;` which I'd recommend adding to clip the text off from the 1px by 1px box we're shoving it into.

---

Thank you - please see e7d1230